### DISCUSSION TITLE
Feature Request: Master Admin

### DISCUSSION THREAD

**Discussion by matbrgz** [outline/outline#1306] [General] [answered]
Is possible to every new collection be add to a admin? Even private collection.

**Comment by tommoor** [comment, marked as answer]
I'm not sure exactly what you mean unfortunately, could you try wording in a different way?

**Comment by matbrgz** [comment]
Let me be clear, administrators are not able to manage the entire Outline. In other words, if a user creates a private collection no one except he can have access. I imagine that a Super Admin with unrestricted access to everything would solve this.
